packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx

```diff
@@ -20,10 +21,16 @@ const ChevronContainer = styled.div`
     width: 30px;
 `;
 
-const ChevronIcon = styled(Icon)`
+const ChevronIcon = styled(Icon)<{ isActive: boolean; canAnimate: boolean }>`
     padding: ${spacingsPx.sm};
     border-radius: 50%;
     transition: background 0.2s;
+
+    ${({ canAnimate, isActive }) =>
+        canAnimate &&
```

Comment:
`canAnimate` is no longer needed

Reply:
Then I was moving it I was doing it first without it but is has to be there, it then animates the wrong way

Reply:
It's already used working [here](https://github.com/trezor/trezor-suite/blob/f02413b6b16088879177c98efde86cfcaa6de2bc/packages/components/src/components/CollapsibleBox/CollapsibleBox.tsx#L82). No need ffor a complex transition ofc but it's literally the same thing, no? Might be wrong!